database_cleaner-ohm.gemspec

```diff
@@ -25,7 +25,14 @@ Gem::Specification.new do |spec|
   spec.add_dependency "database_cleaner-redis", "~> 1.8.0"
   spec.add_dependency "ohm"
 
+  if RUBY_VERSION < '1.9.3'
+    spec.add_development_dependency "rake", "< 11.0.0" # rake 11 requires Ruby 1.9.3 or later
+  elsif RUBY_VERSION < '2.0.0'
+    spec.add_development_dependency "rake", "< 12.0.0" # rake 12 requires Ruby 2.0.0 or later
```

Comment:
@JonRowe @botandrose Do we want to support `database_cleaner` for RUBY_VERSION < 2.0.0? In other gems I've decided to stop maintaining code that is supporting EOLed Ruby versions.

I realize that it can be useful for legacy projects, but as a team of unpaid maintainers I don't think we have the bandwidth for that. 

Would love to read your thoughts. Might deserve a separate issue to discuss. 😄

Reply:
I'm happy with anything that removes these projects from my weekly security emails :joy:

Reply:
I can say that we, RSpec, still maintain back to 1.8.7 because our interpretation of semantic versioning forces that.

Reply:
@JonRowe Yikes! 1.8.7? You have my condolences.

I believe @etagwerker and I had agreed in an earlier PR to database_cleaner core to drop explicit support for EOL Rubies in the v2.0 release.

In my interpretation, what that means concretely is that we only test against Ruby 2.5 and up in CI, but we don't specify `required_ruby_version` in the gemspec. If someone wants to run database_cleaner on Ruby 2.2, and it happens to work, that's great, but its not officially supported, and we probably would be much less inclined to merge PRs that fix bugs on these older Rubies. Also, if we want to drop support for Ruby 2.5 when it goes EOL, we would need to bump database_cleaner to v3.0 in order to do it.

That's how I interpret it, at least. Does that align with your expectations, @etagwerker?

Reply:
@botandrose yeah, I think that's aligned with my expectations.